lib/mjsonwp/errors.js

```diff
@@ -308,6 +308,28 @@ class BadParametersError extends ES6Error {
   }
 }
 
+class ProxyRequestError extends ES6Error {
+  constructor(err, jsonwp) {
+    var message = `Proxy Request Unsuccessful. `;
```

Comment:
This should be `let`. It also does not need to be in the special backtick-enclosed string. Or it can be rewritten as

```js
let message = `Proxy request unsuccessful. ${util.hasValue(jsonwp} ? jsonwp.value : ''}`;
```